homeassistant/components/ee_brightbox/device_tracker.py

```diff
@@ -24,18 +26,65 @@
 })
 
 
-def get_scanner(hass, config):
-    """Return a router scanner instance."""
-    scanner = EEBrightBoxScanner(config[DOMAIN])
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up an EE Brightbox device tracker platform."""
+    scanner = EEBrightBoxScanner(config, add_entities)
+    if not scanner.check_config():
+        _LOGGER.error("No devices found")
+        return
+    scanner.scan_devices()
 
-    return scanner if scanner.check_config() else None
+
+class EEBrightBoxEntity(DeviceTrackerEntity):
+    """Represent an EE Brightbox router entity."""
+
+    def __init__(self, mac, name, scanner):
+        """Set up the entity."""
+        self._attrs = None
+        self._is_connected = None
+        self._mac = mac
+        self._name = name
+        self._scanner = scanner
+
+    @property
+    def device_state_attributes(self):
+        """Return device specific state attributes."""
+        return self._attrs
+
+    @property
+    def is_connected(self):
+        """Return True if device is connected."""
+        return self._is_connected
+
+    @property
+    def mac_address(self):
+        """Return the mac address of the device."""
+        return self._mac
+
+    @property
+    def name(self):
+        """Return the name of the entity."""
+        return self._name
+
+    @property
+    def source_type(self):
+        """Return the source type, eg gps or router, of the device."""
+        return SOURCE_TYPE_ROUTER
+
```

Comment:
Add a `see` method to support the see service. It should set `self._is_connected` to `True` and update `self._attrs` with data attributes from the see service call.